xapian-core/tests/harness/testsuite.cc

```diff
@@ -707,8 +715,15 @@ test_driver::do_run_tests(vector<string>::const_iterator b,
 		    ++res.succeeded;
 		    if (verbose || !use_cr) {
 			out << col_green << " ok" << col_reset << endl;
+
+			if (test_duration.count() >= SLOW_TEST_THRESHOLD) {
+			    out << col_yellow << " SLOW TEST ("
+				<< test_duration.count()
+				<< " ms)" << col_reset << endl;
+			}
 		    } else {
-			out << "\r                                                                               \r";
+			out << "\r                                        "
+			    << "                                   \r";
```

Comment:
We should drop this change when merging, because it doesn't make things better (except that it allows the check script to pass on the diff).

Reply:
I'm happy to ignore the check script when it's not helpful.  The point of it is to automate the checks for things that we see people getting wrong - this takes away some of the duller parts of patch review and the submitter gets initial feedback sooner.

But here if the `<<` between the two string literals is removed then it's totally equivalent to the existing code (adjacent string literals are concatenated early on in parsing), passes the check script, and I think is at least as readable.

Reply:
removed the `<<`.